DEVELOPMENT.md

```diff
@@ -124,6 +124,14 @@ pip install -e ".[gpu]"
 
 ## Testing
 
+
+**Download test data**
+The test data for cleanlab resides in the assets repository. Use the following commands to download test data before running tests.
+```shell
+git clone https://github.com/cleanlab/assets.git
+mv assets/cleanlab_test_data cleanlab/tests/datalab/data
```

Comment:
is this the long-term burden you are imposing on all future developers?

Why is the cleanlab_test_data/ being downloaded into the datalab/data folder?
If this is the long-term plan going forward, shouldn't cleanlab_test_data/ be downloaded into a more universal location that the individual unit-tests then access?

Reply:
This does add an extra step for running tests. I downloaded it into datalab, because that's where all the tests for datalab reside. This data is not required for running tests outside of that directory.

Reply:
It's ok for this PR but this is not very forwards compatible going forward if other tests want to access cleanlab_test_data, so I recommend just copying cleanlab_test_data into: cleanlab/tests/

Also you can greatly improve developer quality-of-life by fetching this file inside the unit-test that requires it, if it does not already exist locally.

Reply:
Okay, I can move it to cleanlab/tests.
I tried fetching it inside the unit test initially, but that was not working quickly for some reason. I'll make a note of it for todos.